PULL_REQUEST_TEMPLATE.md

```diff
@@ -33,6 +33,18 @@ List the APIs or describe the functionality that this PR breaks.
 Workarounds for or expected timeline for deprecation
 
 ## Are documentation updates required?
+<!-- 
```

Comment:
Is this supposed to be part of the markdown? It appears as is in the pull request placeholder.

Reply:
@yknl This is supposed to just serve as a helpful comment for the PR creator. When the PR is submitted, it won't appear in the post, it's only viewable when editing the pull request.
[It's not visible when rendered](https://github.com/blockstack/stacks.js/blob/blockstack/pr-template/PULL_REQUEST_TEMPLATE.md#are-documentation-updates-required) but is [visible when editing](https://raw.githubusercontent.com/blockstack/stacks.js/blockstack/pr-template/PULL_REQUEST_TEMPLATE.md).

Reply:
I'd just go with a consistent template. the implications here are very low, so I'd say we stick to it :)